triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -891,7 +900,13 @@ private[lf] class Runner private (
                   "Trigger rule state updated",
                   "state" -> triggerUserState(state, trigger.defn.level),
                 )
-
+                if (trigger.defn.level == Trigger.Level.High) {
```

Comment:
For low level triggers, we log the entire state so there's no need to log internal state